### ISSUE TITLE
Display country for lower TDWG distributions

### ISSUE THREAD

**Issue by gdower** [CatalogueOfLife/backend#1174]
María Marta gave us feedback that she would like the country to display for lower TDWG levels.

For example, it would be preferable if name was São Paulo, Brazil and Minas Gerais, Brazil:

https://api.checklistbank.org/dataset/9842/taxon/9H2WR/info

**Comment by mdoering** [comment]
Makes sense. Level 4 has as a level 3 parent `Brazil Southeast` though, not the country `Brazil`.
Level 2 then is country `Brazil` for 84. But level 2 can also be `Northern South America` (82), `Caribbean` (81) or `South-Central U.S.A.` (77). I am not sure if we want to render those and cant seem to find a way to include just country names. Thats the whole point of TDWG areas, they are not countries.

All tdwg areas: http://api.checklistbank.org/vocab/area?gazetteer=tdwg

I think ultimately the best would be a small map than can also integrate all other standards like MRGID or ISO codes. It would not be terribly difficult to do so with the GIS features of Postgres.

**Comment by mdoering** [comment]
See https://github.com/CatalogueOfLife/checklistbank/issues/1485
